### ISSUE TITLE
Browser page turns blank

### ISSUE THREAD

**Issue by imronwayne** [Musish/Musish#307]
On many devices (Win10 laptop, Nexus Android tablet, iPhone) and browsers (Edge, Brave, Safari, Chrome, Vivaldi, Opera) interface flashes on screen momentarily and then screen becomes blank. Just began happening this morning, previously worked fine.

**Comment by BrychanOdlum** [comment]
Do you have any reproducible steps for this? Are you able to see any errors in your browser's console when this happens, if so please share.

**Comment by raphaelvigee** [comment]
Also, please make sure to use the "Bug report" issue template in the future as it helps reproducing the error and debug it.

**Comment by imronwayne** [comment]
Yes I see an error (the console messages are below). Some additional
information: when I choose to browse without logging in, I see the Musish
interface as before. This could be why others are not reporting this
issue(?). I'm thinking it's related to my authorization to Apple Music via
the browser/Musish - but that's just a guess on my part.

Console log:

content-tss.js loaded: https://musi.sh/ content-tss.js:2:1
hosted page injected musi.sh:7:9 <https://musi.sh/>
content-ads.js loaded: https://musi.sh/ content-ads.js:2:1
Glitter Drag: Content script is injected by browser successfully
content_script.js:4:1
Glitter Drag: loaded config from storage content_script.js:1151:9
TAB_SIDE has not matched i18n message common.js:337:9
Loading failed for the <script> with source “
https://www.googletagmanager.com/gtag/js?id=UA-131332308-1”. musi.sh:6:1
<https://musi.sh/>
TypeError: "e.relationships is undefined"

   - value https://musi.sh/app.1bdd24c9.js:39
   - value https://musi.sh/app.1bdd24c9.js:39
   - value https://musi.sh/app.1bdd24c9.js:39
   - Ri https://musi.sh/app.1bdd24c9.js:63
   - Pi https://musi.sh/app.1bdd24c9.js:63
   - Ni https://musi.sh/app.1bdd24c9.js:63
   - Ka https://musi.sh/app.1bdd24c9.js:63
   - Va https://musi.sh/app.1bdd24c9.js:63
   - Ru https://musi.sh/app.1bdd24c9.js:63
   - Tu https://musi.sh/app.1bdd24c9.js:63
   - xu https://musi.sh/app.1bdd24c9.js:63
   - Ja https://musi.sh/app.1bdd24c9.js:63
   - enqueueSetState https://musi.sh/app.1bdd24c9.js:63
   - setState https://musi.sh/app.1bdd24c9.js:55
   - e https://musi.sh/app.1bdd24c9.js:39
   - _ https://musi.sh/app.1bdd24c9.js:39
   - _invoke https://musi.sh/app.1bdd24c9.js:39
   - t https://musi.sh/app.1bdd24c9.js:39
   - sf https://musi.sh/app.1bdd24c9.js:39
   - a https://musi.sh/app.1bdd24c9.js:39
   - a https://musi.sh/app.1bdd24c9.js:39
   - j https://musi.sh/app.1bdd24c9.js:39
   - s https://musi.sh/app.1bdd24c9.js:39

app.1bdd24c9.js:63:74894 <https://musi.sh/app.1bdd24c9.js>
Unhandled promise rejection TypeError: "e.relationships is undefined"

   - value https://musi.sh/app.1bdd24c9.js:39
   - value https://musi.sh/app.1bdd24c9.js:39
   - value https://musi.sh/app.1bdd24c9.js:39
   - Ri https://musi.sh/app.1bdd24c9.js:63
   - Pi https://musi.sh/app.1bdd24c9.js:63
   - Ni https://musi.sh/app.1bdd24c9.js:63
   - Ka https://musi.sh/app.1bdd24c9.js:63
   - Va https://musi.sh/app.1bdd24c9.js:63
   - Ru https://musi.sh/app.1bdd24c9.js:63
   - Tu https://musi.sh/app.1bdd24c9.js:63
   - xu https://musi.sh/app.1bdd24c9.js:63
   - Ja https://musi.sh/app.1bdd24c9.js:63
   - enqueueSetState https://musi.sh/app.1bdd24c9.js:63
   - setState https://musi.sh/app.1bdd24c9.js:55
   - e https://musi.sh/app.1bdd24c9.js:39
   - _ https://musi.sh/app.1bdd24c9.js:39
   - _invoke https://musi.sh/app.1bdd24c9.js:39
   - t https://musi.sh/app.1bdd24c9.js:39
   - sf https://musi.sh/app.1bdd24c9.js:39
   - a https://musi.sh/app.1bdd24c9.js:39
   - a https://musi.sh/app.1bdd24c9.js:39
   - j https://musi.sh/app.1bdd24c9.js:39
   - s https://musi.sh/app.1bdd24c9.js:39

app.1bdd24c9.js:39:17527 <https://musi.sh/app.1bdd24c9.js>

   - t https://musi.sh/app.1bdd24c9.js:39
   - exports https://musi.sh/app.1bdd24c9.js:39
   - N https://musi.sh/app.1bdd24c9.js:39
   - exports https://musi.sh/app.1bdd24c9.js:23
   - <anonymous> https://musi.sh/app.1bdd24c9.js:23
   - g https://musi.sh/app.1bdd24c9.js:23
   - b https://musi.sh/app.1bdd24c9.js:23

**Comment by imronwayne** [comment]
Also, see attached video clip of what this looks like in the browser
window. The interface shows for a few seconds and then a white screen.

**Comment by UnimaginaryUnit** [comment]
Same issue.

**Comment by imronwayne** [comment]
Phew! It's not just me. Thanks, UnimaginaryUnit.

**Comment by alonme** [comment]
Was anything done regarding this issue? Anyway we can help?

**Comment by imronwayne** [comment]
Not sure - I had to stop using Musish because of it, and never tried again.

**Comment by alonme** [comment]
@BrychanOdlum Is there anyway i can help solving it? more info? if you have some direction for a fix or a way to start checking where the problem might be - I might be able to contribute

**Comment by alonme** [comment]
@imronwayne Seems like this doesn't happen for me now
